.appveyor.yml

```diff
@@ -63,7 +73,25 @@ before_build:
   - if [%PUBLISH_BINARY%]==[true] set PATH=C:\stellarium-OpenSSL;%PATH%
   - if [%PUBLISH_BINARY%]==[true] appveyor DownloadFile https://github.com/Stellarium/stellarium-data/releases/download/qt-5.6/%SSL%Win%BITS%.zip -FileName c:\stellarium-OpenSSL.zip
   - if [%PUBLISH_BINARY%]==[true] 7z e c:\stellarium-OpenSSL.zip -aoa -oc:\stellarium-OpenSSL
-  - mkdir stellarium-%qtver%-%qtbin%
```

Comment:
Why this line is removed?

Reply:
Because it creates a directory in the current directory, which has changed by some `cd` commands above. And editing it to an absolute path didn't make sense to me, because it will be created by CMake's `install` target anyway.